.github/workflows/on-pr-closed.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    branches:
+      - master
+    types: ["closed"]
```

Comment:
will this be called if the pr was closed but not merged? 
should we try something mentioned in https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges-1

Reply:
Ah good catch, yes let's do that

Reply:
Ok updated the PR @dahuang-purestorage. Tested it here too:
https://github.com/ggriffiths/openstorage/pull/18

First, the action was skipped when it closed
Second, it ran when I reopened and merged the PR.